be more explicit that VC examples do not define anything

bc-pi opened the pull request:
be more explicit that the VCDM and SD-JWT VC examples are only illustrative and do not define anything (for #341)

preview links to the two relevant appendix parts: 
https://drafts.oauth.net/oauth-selective-disclosure-jwt/examples-are-only-examples/draft-ietf-oauth-selective-disclosure-jwt.html#appendix-A

https://drafts.oauth.net/oauth-selective-disclosure-jwt/examples-are-only-examples/draft-ietf-oauth-selective-disclosure-jwt.html#appendix-A.4

Sakurann commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1567,9 +1567,10 @@ After the validation, the Verifier will have the following data for further proc
-SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
-Person Identification Data (PID) defined in [@EUDIW.ARF] is used.
+This example shows how the artifacts defined in this specification
+could hypothetically be used to express an
```
```suggestion
could be used to express an
```

Sakurann commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1604,8 +1605,8 @@ After the validation, the Verifier will have the following data for further proc
 
-In this example, the artifacts defined in this specification are used to represent a payload
-that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0].
+This hypothetical example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
```
```suggestion
This example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
```

Sakurann commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1604,8 +1605,8 @@ After the validation, the Verifier will have the following data for further proc
 
-In this example, the artifacts defined in this specification are used to represent a payload
-that is represented as a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0].
+This hypothetical example shows how a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0]
+payload might be expressed using the artifacts defined in this specification.
```
```suggestion
This is a non-normative example how the artifacts defined in this specification
could hypothetically be used to express
a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] payload.
```

bc-pi replied:
riffing on @Sakurann's suggestion 
```suggestion
This is non-normative example illustrates how the artifacts defined in this specification
could be used to express a W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] payload.
```

Sakurann commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1567,9 +1567,10 @@ After the validation, the Verifier will have the following data for further proc
-In this example, the artifacts defined in this specification are used to represent
-SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
-Person Identification Data (PID) defined in [@EUDIW.ARF] is used.
+This example shows how the artifacts defined in this specification
+could hypothetically be used to express an
+SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
+Person Identification Data (PID) defined in [@EUDIW.ARF].
```
```suggestion
This is a non-normative example how the artifacts defined in this specification
could hypothetically be used to express an
SD-JWT-based Verifiable Credential [@I-D.ietf-oauth-sd-jwt-vc] with
Person Identification Data (PID) defined in [@EUDIW.ARF].
```

Sakurann replied:
```suggestion
This example shows how the artifacts defined in this specification could be used to represent
SD-JWT-based Verifiable Credentials (SD-JWT VC) as defined in [@I-D.ietf-oauth-sd-jwt-vc].
Person Identification Data (PID) defined in [@EUDIW.ARF] is used as a sample payload.
```

Sakurann replied:
ok, so I think editors discussed to merge #407 first, and not cover in this PR?

bc-pi replied:
#407 was merged and addresses this part

Sakurann commented:
discussed to add a sentence at line 1513 that data structure and claim names are hypothetical.

bc-pi commented:
> discussed to add a sentence at line 1513 that data structure and claim names are hypothetical. @Sakurann to do

Text from SD-JWT VC that tries to say similar things:
> Important: The following examples are not normative and provided for
> illustration purposes only. In particular, neither the structure of the claims
> nor the selection of selectively disclosable claims are normative.

bc-pi commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1713,8 +1713,9 @@ data. The original JSON data is then used by the application. See
+* Be more explicit that the VCDM and SD-JWT VC examples are only illustrative and do not define anything
 * Remove mention of unspecified key binding methods and the Enveloping SD-JWTs section
-
+* Update PID example
```
I just happened to notice this was in the wrong draft revision when resolving conflicts merging in master

danielfett replied:
Thanks, I seem to be bad at putting changelog entries in the right place.

bc-pi replied:
Maybe because I indented -08 differently than the others? It's my fault...

bc-pi commented:
> > discussed to add a sentence at line 1513 that data structure and claim names are hypothetical. @Sakurann to do
> 
> Perspective text from SD-JWT VC that tries to say similar things:
> 
> > Important: The following examples are not normative and provided for
> > ~illustration~ illustrative purposes only. In particular, neither the structure of the claims
> > nor the selection of selectively disclosable claims are normative.

I added that text at start of the whole examples appendix 0bd026ba5f802a79c790822f875ea348647fb11c

And updated the text at the start of the VCDM example along the lines of @Sakurann's suggestion 30c847c605fd06064ade786ff9c3f134b9ff32af 

I think this one is ready for review again.

Sakurann reviewed (approved):
minor suggestion, but LGTM - thank you.

Sakurann commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1456,7 +1456,11 @@ the media type is encoded as an SD-JWT.
-All of the following examples are non-normative.
+Important: The following examples are not normative and provided for
+illustrative purposes only. In particular, neither the structure of the claims
+nor the selection of selectively disclosable claims are normative.
```
```suggestion
nor the selection of selectively disclosable claims is normative.
```

neither ... or ... is ... . I think

bc-pi replied:
is it is

bc-pi commented on draft-ietf-oauth-selective-disclosure-jwt.md:
```diff
@@ -1456,7 +1456,11 @@ the media type is encoded as an SD-JWT.
 # Additional Examples
 
-All of the following examples are non-normative.
+Important: The following examples are not normative and provided for
```
```suggestion
Important: The following examples are not normative and are provided for
```

bc-pi commented:
The only changes Dr. @danielfett hasn't seen since he approved are text he wrote/suggested borrowed from SD-JWT VC so I think it's ok to go ahead and merge.
